index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+     
+      <div class="menu" id="menu">
+        <a class="menu-icon" href="#menu">
+            <img src="./assets/images/Enabled.svg" alt="" srcset="">
+          </a>
```

Comment:
- [ ] Kindly wrap the hamurger icon or the menu image inside a `nav` tag , as requested in the requieremnts 👍 

```bash
<nav>
      <a>...</a>
       ...
</nav>
```